### DISCUSSION TITLE
Igualdade entre defrecord e mapas

### DISCUSSION THREAD

**Discussion by wandersoncferreira** [clj-br/forum#5] [Perguntas e Respostas] [answered]
Pessoal, eu posso interpretar uma instância de `defrecord` como um mapa em várias situações. Por que esse resultado abaixo é falso?

![Screenshot from 2021-06-27 19-48-22](https://user-images.githubusercontent.com/17708295/123561780-ca9cee00-d780-11eb-9b12-2dc9fea7010f.png)

EDIT: Como eu faria para que essa igualdade retornasse `true`?

**Comment by ericdallo** [comment, marked as answer]
Acredito que seja porque são classes diferentes por traz dos panos, algo como `(.equals (Foo. 1) {:a 1})` deve funcionar.
